backend/src/seed/seed-db.js

```diff
@@ -123,30 +113,16 @@ import Factory from './factories'
     ])
 
     await Promise.all([
-      f.relate('User', 'Badges', {
-        from: 'b6',
-        to: 'u1',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b5',
-        to: 'u2',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b4',
-        to: 'u3',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b3',
-        to: 'u4',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b2',
-        to: 'u5',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b1',
-        to: 'u6',
-      }),
+      peterLustig.relateTo(racoon, 'rewarded'),
+      peterLustig.relateTo(rhino, 'rewarded'),
+      peterLustig.relateTo(wolf, 'rewarded'),
+      bobDerBaumeister.relateTo(racoon, 'rewarded'),
+      bobDerBaumeister.relateTo(turtle, 'rewarded'),
+      jennyRostock.relateTo(bear, 'rewarded'),
+      dagobert.relateTo(rabbit, 'rewarded'),
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-11T14:47:16Z" title="Thursday, July 11th 2019, 4:47:16 pm +02:00">Jul 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -123,30 +113,16 @@ import Factory from './factories'
     ])
 
     await Promise.all([
-      f.relate('User', 'Badges', {
-        from: 'b6',
-        to: 'u1',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b5',
-        to: 'u2',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b4',
-        to: 'u3',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b3',
-        to: 'u4',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b2',
-        to: 'u5',
-      }),
-      f.relate('User', 'Badges', {
-        from: 'b1',
-        to: 'u6',
-      }),
+      peterLustig.relateTo(racoon, 'rewarded'),
+      peterLustig.relateTo(rhino, 'rewarded'),
+      peterLustig.relateTo(wolf, 'rewarded'),
+      bobDerBaumeister.relateTo(racoon, 'rewarded'),
+      bobDerBaumeister.relateTo(turtle, 'rewarded'),
+      jennyRostock.relateTo(bear, 'rewarded'),
+      dagobert.relateTo(rabbit, 'rewarded'),
```


oooo, now we have `dagobert`, interesting

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-11T19:57:10Z" title="Thursday, July 11th 2019, 9:57:10 pm +02:00">Jul 11, 2019</time>_

---


We had him before, we just didn't assign him to a variable.